react/src/app.jsx

```diff
@@ -14,24 +15,29 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    this.fetch();
+    // this.fetch();
```

Comment:
What prompted the change from ajax to axios? Just preference?

Reply:
Axios is just a much better features and in my opinion better overall system than what jQuery has to offer.